lib/rules/web/admin_console/4.3/common_ui_elements.xyaml

```diff
@@ -91,14 +97,18 @@ check_row_filter_on_page:
 confirm_deletion:
   action: send_delete_string
   action: submit_changes
+check_resource_item:
```

Comment:
This change was already in #560 , temporally add here will rebase after #560 gets merged

Reply:
already rebased